elide-core/src/main/java/com/yahoo/elide/utils/coerse/CoerceUtil.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2015, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.utils.coerse;
+
+import com.yahoo.elide.core.exceptions.InvalidAttributeException;
+import com.yahoo.elide.utils.coerse.converters.ToEnumConverter;
+import org.apache.commons.beanutils.*;
+
+/**
+ * Class for coercing a value to a target class
+ */
+public class CoerceUtil {
+
+    private static final ToEnumConverter ENUM_CONVERTER = new ToEnumConverter();
+
+    //static block for setup and registering new converters
+    static {
+        setup();
+    }
+
+    /**
+     * Convert value to target class
+     * @param value value to convert
+     * @param cls class to convert to
+     * @return
+     */
+    public static Object coerce(Object value, Class<?> cls) {
+
+        if (value == null || cls == null || cls.isAssignableFrom(value.getClass())) {
+            return value;
+        }
+
+        try {
+            return ConvertUtils.convert(value, cls);
+
+        } catch (ConversionException | InvalidAttributeException e) {
+            throw new IllegalArgumentException("Unable to coerce " + value.getClass() + " to " + value);
```

Comment:
Should we throw a checked exception here or let this bubble up? Does this result in a 500 error from the server if we hit this?